packages/updating-element/src/updating-element.ts

```diff
@@ -534,6 +537,26 @@ export abstract class UpdatingElement extends HTMLElement {
    */
   private _reflectingProperty: PropertyKey | null = null;
 
+  /**
+   * Set of callbacks called in connectedCallback.
+   */
+  connectedCallbacks: Set<connectCallback> = new Set();
```

Comment:
I'm not sure sets really buy us anything here. If the pattern is that a controller accepts a host in its constructor, we're very unlikely to to get duplicate callbacks. If we do, that's on the controller.

Reply:
The set buys efficient removal and less code for doing so.